src/Event/Http/FpmHandler.php

```diff
@@ -108,16 +113,19 @@ public function __destruct()
      */
     public function handleRequest(HttpRequestEvent $event, Context $context): HttpResponse
     {
+        $this->lambdaRequestId = $context->getAwsRequestId();
```

Comment:
Why are we mutating the handler state?

Reply:
Yes, this is exactly where I stumbled when working on this locally.

I don't really want to store state in the class, but then that means passing it everywhere.

This led me to realize that in most cases there is no context associated to the log lines, since we boot PHP-FPM _before_ any request comes in.

In the end, I'm not 100% sure about the approach, and #895 may be able to remove the source issue.

Reply:
Yeh, I agree #895 or something like it, is really the best option here.